drf_integrations/forms.py

```diff
@@ -58,7 +58,7 @@ def __init__(self, data=None, *args, **kwargs):
         else:
             # Application has no integration
             # Render config field
-            self.fields["config"] = JSONField()
+            self.fields["config"] = JSONField(required=False)
```

Comment:
What about setting `default=dict` instead of `required=False` - that way we have a dictionary there (how we use it now) instead of opening it up to interpretation?